lesion_segmentation_protocol.md

```diff
@@ -0,0 +1,72 @@
+
+```json
+{
+        "GeneratedBy": [
+            {
+                "Name": "2D nnUNet model model_ms_seg_sc-lesion_regionBased.zip",
+                "Version": "https://github.com/ivadomed/canproco/releases/tag/r20240125",
+                "Date": "2024-01-26"
+            }
+        ]
+    }
+```
+
+- To manually create/correct the segmentation, please use the manual-correction (https://github.com/spinalcordtoolbox/manual-correction) repository. The command can be inspired by this: 
+
+```console
+python manual_correction.py -path-img ~/data/canproco -config ~/config_seg.yml  -path-label ~/data/canproco/derivatives/labels  -suffix-files-lesion _lesion-manual -fsleyes-dr="-40,70"
+``` 
+
+- A Quality Control (QC) report should be produced using SCT, and added to a GitHub issue for further validation by other investigators. Using SCT, you can review lesion segmentation in the axial or sagittal plane:
+
+```console
+sct_qc -i {image_file} -d {lesion_seg_file} -s {sc_seg_file} -p sct_deepseg_lesion -plane {sagittal, axial} -qc {canproco_qc_folder}
+```  
+
+- If you are not sure about the segmentation on a subject, it should be flagged on GitHub for a more open discussion: here are some examples [(1)](https://github.com/ivadomed/ms-lesion-agnostic/issues/4#issuecomment-1947326493) and [(2)](https://github.com/ivadomed/ms-lesion-agnostic/issues/4#issuecomment-1947338624)
+
+## Step 1: Software to create/correct lesions
+It is common practice to use FSLeyes at NeuroPoly for visual inspection of MRI images and manual segmentation of MS lesions. Therefore, naturally, the first step of the lesion segmentation process is to complete the FSLeyes tutorial ([FSLeyes documentation](https://open.win.ox.ac.uk/pages/fsl/fsleyes/fsleyes/userdoc/) and [video tutorial](https://www.youtube.com/playlist?list=PLIQIswOrUH69qFMNg8KYkEGkvCNEwlnfT)). Trainees are encouraged to learn keyboard shortcuts (ctrl+F to toggle an image, shift+↑ to scroll through volumes, ...).
+
+Furthermore, it is recommended to get familiar with SCT for creating QCs and for manual correction ([SCT tutorial](https://spinalcordtoolbox.com/user_section/tutorials.html)). 
+
+## Step 2: Spinal cord anatomy and lesion segmentation
+Before, moving on to MS lesion segmentation, trainees are advised to study the neuroanatomical structures of healthy spinal cords. Trainees should look at healthy spinal cords in MRI images of different contrasts: T2w, T1w, PSIR, STIR, MP2RAGE... A public dataset will be built for this purpose.
+
+To learn the specificity of MS lesions, trainees should work on differentiating MS lesions from other diseases. 
+
+One of the most challenging tasks of MS lesion segmentation is to distinguish the border of a lesion and the cerebrospinal fluid (CSF). To learn where to draw the lesion border, a set of tricky examples validated by a NeuroRadioligist will be created. 
```

Comment:
Reference the videos made by Julien during work with Michelle Chen regarding MS lesion segmentation